targets/mvebu-cortexa53

```diff
@@ -0,0 +1,5 @@
+device('gl.inet-gl-mv1000', 'glinet_gl-mv1000', {
+	factory = false,
+	sysupgrade = '-ext4-sdcard',
```

Comment:
@herbetom do you mean like so?

```suggestion
	sysupgrade = '-squashfs-sdcard',
```

was me first time creating a new image for the community for us so therefore, I am grateful for any suggestions for improvement

Reply:
yes, no worries :)

Obviously needs to be tested but if that works it's probably the way to go.

Reply:
as soon as the image build i guess the person in our community can test the build again